nx.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "./node_modules/nx/schemas/nx-schema.json",
+  "targetDefaults": {
+    "build": {
+      "dependsOn": ["^build"],
+      "inputs": [
+        "{projectRoot}/src/**/*",
+        "{projectRoot}/tsconfig.json",
+        "{projectRoot}/package.json",
+        "{projectRoot}/modern.config.*",
+        "{projectRoot}/scripts/**/*"
+      ]
+    }
+  },
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "nx-cloud",
```

Comment:
Will we pay for nx-cloud?

Reply:
I aim to utilize nx cloud primarily to determine the causes of cache misses, rather than for remote caching. Consequently, I think it may not necessitate any payment.